### PR TITLE
release: version 2.13.0

### PR DESCRIPTION
**Pull request by sivanharel**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_528/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+### Features
+
+* **Button:** Use smart contrast on hover ([7eecbb0](https://github.com/wix/wix-ui-tpa/commit/7eecbb024534bab827650884a99e782e1c4721a5))
+* **RadioButton, Checkbox:** Add disabled mode state for theme box and some bug fixes. ([7e81f40](https://github.com/wix/wix-ui-tpa/commit/7e81f40bbe8311df5610c3fea4ffe4550c156d85))
+* **DatePickerInput:** - Add for working in progress ([2847526](https://github.com/wix/wix-ui-tpa/commit/2847526b051f126ec0afab3142f883e2c43a7a72))
+
+### Reverts
+
+* "fix(Button): Fix prefix and suffix alignments" ([8a18c7b](https://github.com/wix/wix-ui-tpa/commit/8a18c7b6edb547d7849db63cde2fd8bf98e68728))
```
Comment (nitayneeman) [inline CHANGELOG.md:18]: Maybe it should be under "Bug Fixes"?

**Review by nitayneeman — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jonathanadler) [reply CHANGELOG.md:18]: 🤔  I think it's more accurate to use "revert" here, since this is a revert of a bug fix.
So it's a fix of a bug fix 😄 .

Comment (jonathanadler) [reply CHANGELOG.md:18]: I used the "revert" commit message as defined here: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#revert-commits

**Pull request merged**